docs/src/main/paradox/release-notes/releases-1.1.md

```diff
@@ -99,6 +103,7 @@ Most of the dependency changes are small patch level upgrades. Some exceptions i
 
 ### Known Issues
 
+* A race condition has been introduced into pekko-cluster-sharding which needs to be fixed in a Pekko 1.1.1 release. Users who use Cluster Sharding should skip this release. The problem does not appear in v1.1.0-M1. ([#1463](https://github.com/apache/pekko/pull/1463))
```

Comment:
It also can be temporarily fixed by using another API in classic ClusterSharding(can not fix on typed pekko):  https://github.com/apache/pekko/blob/09486097e7115a0a173faf2fca929f9d62f80394/cluster-sharding/src/main/scala/org/apache/pekko/cluster/sharding/ClusterSharding.scala#L267-L273

Reply:
We should have 1.1.1 released in a couple of days so it is probably not very important to put this is in the docs but if anyone gets in touch in the mean time, then we can share this info with them. Thanks.